web/src/lib/utils/asset-utils.ts

```diff
@@ -111,3 +111,38 @@ export async function bulkDownload(
 		});
 	}
 }
+
+/**
+ * Returns the filename extension without a dot (.) and an empty
+ * string when not found.
+ */
+export function getFilenameExtension(filename: string): string {
+	const lastIndex = filename.lastIndexOf('.');
+	return filename.slice(lastIndex + 1);
```

Comment:
should we make this `toLowerCase` for uniformity?

Reply:
Yes definitely, because uploading files with uppercase file extensions would otherwise fail. I just committed an update to fix this.